app/src/main/java/org/mozilla/fenix/onboarding/HomeCFRPresenter.kt

```diff
@@ -38,14 +38,16 @@ class HomeCFRPresenter(
      * Determine the CFR to be shown on the Home screen and show a CFR for the resultant view
      * if any.
      */
-    fun show() {
-        when (val result = getCFRToShow()) {
+    fun show(): Boolean {
+        val result = getCFRToShow()
+        when (result) {
             is Result.SyncedTab -> showSyncedTabCFR(view = result.view)
             is Result.JumpBackIn -> showJumpBackInCFR(view = result.view)
             else -> {
                 // no-op
             }
         }
+        return result != Result.None
```

Comment:
```suggestion
        return when (val result = getCFRToShow()) {
            is Result.SyncedTab -> { 
               showSyncedTabCFR(view = result.view)
               true
            }
            is Result.JumpBackIn -> {
                showJumpBackInCFR(view = result.view)
                true
            }
            else -> false
        }
```
I would also consider writing it like this so we can get rid of the no-op